src/dotnet/APIView/APIViewUITests/SmokeTests.cs

```diff
@@ -172,7 +172,7 @@ public void SmokeTest_CSharp()
             }
         }
 
-        [Fact]
+        [Fact(Skip = "Test is failing to find element")]
```

Comment:
Lets be sure to file an issue for re-enabling this

Reply:
https://github.com/Azure/azure-sdk-tools/issues/7329